src/main/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialect.java

```diff
@@ -239,11 +255,101 @@ protected String getSqlType(SinkRecordField field) {
     }
   }
 
+  @Override
+  protected ColumnDefinition columnDefinition(
```

Comment:
What is the significance of this override?

Reply:
I will remove this override. I think it was leftover from a previous approach that didn't work out.